src/db/tables/subscribers.ts

```diff
@@ -350,17 +345,8 @@ async function getPotentialSubscribersWaitingForFirstDataBrokerRemovalFixedEmail
     // ...who haven't received the email...
     .andWhere("first_broker_removal_email_sent", false)
     // ...and signed up after the feature flag `FirstDataBrokerRemovalFixedEmail`
-    // has been enabled last.
-    .andWhere("created_at", ">=", flag.modified_at);
```

Comment:
@flozia From [this comment](https://mozilla-hub.atlassian.net/browse/MNTOR-2700?focusedCommentId=907914), I think it was around this time that this flag was enabled, and if I'm understanding the query correctly, as long as this date isn't _earlier_ than that the flag was enabled, we should be good, right? I could also move it some more into the future, if we want to be safe.

Reply:
@Vinnl Yes that’s right.